src/Microsoft.ML.FastTree/RandomForestRegression.cs

```diff
@@ -209,6 +211,17 @@ private static FastForestRegressionModelParameters Create(IHostEnvironment env,
 
         private protected override PredictionKind PredictionKind => PredictionKind.Regression;
 
+        bool ISingleCanSaveOnnx.SaveAsOnnx(OnnxContext ctx, string[] outputNames, string featureColumn)
+        {
+            // Mapping score to prediction
+            var fastTreeOutput = ctx.AddIntermediateVariable(null, "FastTreeOutput", true);
+            var numTrees = ctx.AddInitializer((float)TrainedEnsemble.NumTrees, "NumTrees");
+            base.SaveAsOnnx(ctx, new[] { fastTreeOutput }, featureColumn);
```

Comment:
NumTrees is an integer variable. And AddInitializer supports long as a parameter. Can you please explain why it is necessary to cast it to float?

Reply:
The onnx "Div" operation requires that both operands are of the same type T, and FastTreeOutput has to be a float.